### PR TITLE
Amiram first Feedback done plese check

### PR DESCRIPTION
**Pull request by AmiramAbergel**

### CODE REVIEW

File: home_assignment/controllers/messages_routes.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+def router(app):
+    # @app.route("/example", methods=["GET"])
+    # def example():
+    #     return json_dumps({"This Is": "An example"})
+    #
+    # @app.route("/do", methods=["POST"])
+    # def process_application():
+    #     request_data = json_loads(request.get_json())
+    #     my_param_key = request_data["my_param_key"]
+    #     return json_dumps({"message": f"Done with {my_param_key}"})
```
Comment (tomsender) [inline home_assignment/controllers/messages_routes.py:19]: remove

File: home_assignment/controllers/messages_routes.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        request_data = request.get_data()
+        data = json.loads(request_data)
+        user_id = data.get('user_id')
+        name = data.get('name')
+        title = data.get('title')
+        message = data.get('message')
+        read = False
+        created_at = data.get('created_at')
+
+        flow = SendMessageFlow("Test1")
```
Comment (tomsender) [inline home_assignment/controllers/messages_routes.py:32]: test1?

File: home_assignment/dal/inmemory_database.py [hunk 3]
```diff
@@ -14,23 +15,39 @@ def add(self, message: Message):
...
+        title = message[0].title
+        user_message = message[0].message
+        read = message[0].read
+        created_at = message[0].created_at
+        return {f"id: {user_id}, "
+                               f"name: {name}, "
+                               f"title: {title}, "
+                               f"message: {user_message}, "
+                               f"read: {read}, "
+                               f"created_at: {created_at} "
```
Comment (tomsender) [inline home_assignment/dal/inmemory_database.py:33]: Why do you use fstring here? you build a dict

File: home_assignment/controllers/messages_routes.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        result = flow.send(user_id, name, title, message, read, created_at)
+        if user_id and name and title and message:
+            return {"New message": f"id: {result.user_id}, "
+                                   f"name: {result.name}, "
+                                   f"title: {result.title}, "
+                                   f"message: {result.message}, "
+                                   f"read: {result.read}, "
+                                   f"created_at: {result.created_at} "
```
Comment (tomsender) [inline home_assignment/controllers/messages_routes.py:40]: Think about extendability and SOLID.
Why do you need to write it again and again?

**Review by tomsender — COMMENTED** [review]
Great job!

File: home_assignment/controllers/messages_routes.py (hunk 1 above)
Comment (AmiramAbergel) [reply home_assignment/controllers/messages_routes.py:19]: done.

File: home_assignment/controllers/messages_routes.py (hunk 2 above)
Comment (AmiramAbergel) [reply home_assignment/controllers/messages_routes.py:32]: fixed.

File: home_assignment/dal/inmemory_database.py (hunk 3 above)
Comment (AmiramAbergel) [reply home_assignment/dal/inmemory_database.py:33]: Replaced with dict

File: home_assignment/controllers/messages_routes.py (hunk 4 above)
Comment (AmiramAbergel) [reply home_assignment/controllers/messages_routes.py:40]: DONE.

**Pull request closed without merging**